__tests__/sources.ts

```diff
@@ -1859,6 +1859,22 @@ describe('mutation createSquad', () => {
     );
   });
 
+  it('Should throw error when post moderation is required, but posting is restricted to moderators only', async () => {
+    loggedUser = '1';
+    return testMutationErrorCode(
+      client,
+      {
+        mutation: MUTATION,
+        variables: {
+          ...variables,
+          moderationRequired: true,
+          memberPostingRole: SourceMemberRoles.Moderator,
+        },
+      },
+      'GRAPHQL_VALIDATION_FAILED',
```

Comment:
Since other cases can throw `GRAPHQL_VALIDATION_FAILED` can you use `testMutationError` to check the error message specifically in addition to code, so we are sure correct error is returned?

Reply:
Done!